doc/style.md

```diff
@@ -5,6 +5,8 @@ All of these are style recommendations, no hard rules.
 ## Spec
 
 Be as general as possible, while still being correct.
+
+* Prefer named specs over predicates e.g. `::ifn` instead of `ifn?` [(#76)](https://github.com/slipset/speculative/issues/76)
```

Comment:
I only put this first because later points about using various predicates should be in the context of this point.